pupynere.py

```diff
@@ -989,6 +1003,9 @@ def itemsize(self):
     itemsize = property(itemsize)
 
     def __getitem__(self, index):
+        
+        if self.data is None:
```

Comment:
Would this be better as `if not self.data_allocated()`? More robust, possibly slightly clearer?

Reply:
Good idea, thanks. Will fix.